src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -172,6 +172,11 @@ protected internal void SetStyleArranged(params (string name, string value)[] st
 #endif
 		}
 
+		protected internal void SetNativeTransform(double scaleX, double scaleY, double translateX, double translateY)
```

Comment:
`Transform` on WASM are not using this (and cannot since they are using `matrix`) ... we should either change the name of this method (it seems to be used only by shapes to ensure the stretching), either accept a `matrix` as input and make the `Transform` use it.

Reply:
Indeed, I'll take a look.

Reply:
You should look at here :) : https://github.com/nventive/Uno/blob/f93828c65af3584929b27183f2e9641773d0f09d/src/Uno.UI/Media/NativeRenderTransformAdapter.wasm.cs#L29

Reply:
There you go :) Should be a lot better indeed.